packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/extension.tsp

```diff
@@ -0,0 +1,174 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+namespace Azure.ResourceManager.Models.Resources;
+
+@parentResource(TopLevelTrackedResource)
+model CustomExtensionResource is ExtensionResource<CustomExtensionResourceProperties> {
+  ...ResourceNameParameter<CustomExtensionResource>;
+}
+
+/** CustomExtensionResource properties */
+model CustomExtensionResourceProperties {
+  @doc("The description of the resource.")
+  description?: string;
+
+  /** The status of the last operation. */
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+}
+
+@armResourceOperations
+interface CustomExtensionResources {
```

Comment:
Change to `ExtensionResources`.

Reply:
Fixed in the new version https://github.com/Azure/cadl-ranch/pull/701/commits/3399f26b6ea8081e0519ef28c9521392e0769724